src/constants/constants.h

```diff
@@ -2,7 +2,7 @@
 #define CONSTANTS_H
 
 #define HOST_IP "localhost"
-#define HOST_PORT 9090
+#define HOST_PORT 9000
```

Comment:
Remnant of testing? This should be changed back to `9090`.

Reply:
Whoops yeah